examples/README.rst

```diff
@@ -13,3 +13,14 @@ series (magnitude and phase images) and respiratory trace information obtained f
 multi-gradient echo (MGRE) magnitude image is used to generate an ROI and resample the static and dynamic Gx, Gy, Gz
 component maps to match the MGRE image. Lastly the average Gx, Gy, Gz values within the ROI are computed for each
 slice.
+
+demo_b1_shimming.sh
+--------------------------
+
+This  `shell script <https://github.com/shimming-toolbox/shimming-toolbox/blob/master/examples/demo_b1_shimming.sh>`__
+calls different shimming-toolbox command lines functions to perform a B1+ shimming scenario. Text files, scaled B1 maps
+and results figures are generated in the `testing-data` folder.
+
+This script shows how to perform a B1+ shimming experiment using the command lines functions. In this example, a
+magnitude least square algorithm is used to target a B1+ value specified by the user. This scenario assumes that the
+B1 maps have already been converted into NIfTI files.
```

Comment:
could you add some figures to show what the user is expected to see? this feel a bit abstracts and it is unclear what files users need, what sequence, etc. A lot of background is missing here to be able to understand how to use this script.

Reply:
Will do in #326